app/assets/stylesheets/resource_curriculum_show.scss

```diff
@@ -1,42 +1,45 @@
 @import 'color';
 
-
-.resources_container a, .curriculums_container a {
-  color: $brick;
-  &:hover {
-    color: lighten( $brick, 10% );
-    text-decoration: underline;
+.resources_container, .curriculums_container {
```

Comment:
Selector `resources_container` should be written in lowercase with hyphens<br>Selector `curriculums_container` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line